WNPRC_EHR/test/src/org/labkey/test/tests/wnprc_ehr/WNPRC_EHRTest.java

```diff
@@ -2305,10 +2306,9 @@ public void testEnterWeights() throws IOException, CommandException
     {
         navigateToWeights();
         fillWeightForm(WEIGHT_VAL.toString(),0);
-        waitUntilElementIsClickable("submit-all-btn");
-        //shortWait().until(ExpectedConditions.elementToBeClickable(Locator.id("submit-all-btn")));
         clickNewButton("submit-all-btn");
-        clickAndWait(Locator.tagWithId("button","submit-final"));
+        ModalDialog modalDialog = new ModalDialog.ModalDialogFinder(getDriver()).waitFor();
+        modalDialog.dismiss("Submit final");
```

Comment:
The previous code was waiting for a page load. The new code should continue to do so.
```suggestion
        ModalDialog modalDialog = new ModalDialog.ModalDialogFinder(getDriver()).waitFor();
        doAndWaitForPageToLoad(() -> modalDialog.dismiss("Submit final"));
```
So should other places that click `submit-final`, I would presume.